docs/guidance/contracts-to-projects.md

```diff
@@ -6,25 +6,25 @@ An increasing number of procurement portals now publish data using the Open Cont
 
 * Every update to that process, from planning through to implementation, should be published under the same `ocid`, and in a structured open data format;
 
-* It should be possible to download bulk data in OCDS format, or access this structured data via an API. 
+* It should be possible to download bulk data in OCDS format, or access this structured data via an API.
 
 Even when an OCDS publisher does not provide data for every stage of the contracting process, it is still possible to use OCDS data to:
 
 * Discover contracts related to infrastructure projects;
 
-* Track these contracting processes, including changes to tenders, details of suppliers selected, and, in some cases, details of contract variations. 
+* Track these contracting processes, including changes to tenders, details of suppliers selected, and, in some cases, details of contract variations.
 
 ### Getting started
 
 *The following steps may require support from a technical expert. You can also contact the OCDS Helpdesk ([data@open-contracting.org](mailto:data@open-contracting.org)) for guidance.*
 
 #### (1) Evaluate the Open Contracting Data
 
-Check that the data you plan to analyse is in OCDS format (Tip: you can use the [data validator/review tool](http://standard.open-contacting.org/validator) for this). 
+Check that the data you plan to analyse is in OCDS format (Tip: you can use the [data validator/review tool](http://standard.open-contacting.org/validator) for this).
```

Comment:
Update the name and link.

Reply:
Addressed in https://github.com/open-contracting/infrastructure/pull/46/commits/5c760acb02f31f25ee19ea29362dc314da9f633c